modelbaker/dbconnector/db_connector.py

```diff
@@ -234,7 +236,12 @@ def get_ignored_layers(self, ignore_basket_tables=True):
                     detected_tables.append(record["tablename"])
                     continue
             if "schemaname" in record:
-                if record["schemaname"] in IGNORED_SCHEMAS:
+                ignored = False
+                for ignored_schema_pattern in IGNORED_SCHEMAS:
+                    if fnmatch.fnmatch(record["schemaname"], ignored_schema_pattern):
```

Comment:
Need to check if record["schemaname"] is None?

Reply:
Ah gpkgconnector writes null as schemaname. Well, I don't question that right now.